systest/scripts/tempest_config.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+# Copyright 2017 F5 Networks Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+set -ex
+
+# Copy over our default tempest files
+cp conf/tempest.conf ${TEMPEST_CONFIG_DIR}/tempest.conf.orig
```

Comment:
I don't think it was taken out because we need it for the sed part of the script. We copy it over as .orig and then the sed puts the new output in the real name.

```
cat ${TEMPEST_CONFIG_DIR}/tempest.conf.orig \
  | sed "s/{{ OS_CONTROLLER_IP }}/${OS_CONTROLLER_IP}/" \
  | sed "s/{{ OS_PUBLIC_ROUTER_ID }}/${OS_PUBLIC_ROUTER_ID}/" \
  | sed "s/{{ OS_PUBLIC_NETWORK_ID }}/${OS_PUBLIC_NETWORK_ID}/" \
  | sed "s/{{ OS_CIRROS_IMAGE_ID }}/${OS_CIRROS_IMAGE_ID}/" \
  > ${TEMPEST_CONFIG_DIR}/tempest.conf
```